README.md

```diff
@@ -44,6 +44,7 @@
 | :------------: |:-------------: |:-----:|
 | New Snippet    | `Cmd/Ctrl + N` | Create a snippet      |
 | Edit Snippet   | `Cmd/Ctrl + E` | Edit a snippet      |
+| Delete Snippet   | `Del` | Delete selected snippet      |
```

Comment:
Use `Cmd/Ctro + Del` instead to avoid the users accidentally deleting the gist by hitting 'del'?

Reply:
@hackjutsu Thought about this but because we have a confirmation modal after hitting the 'Del' key that requires a mouse click on the "Delete" button to actually do this action I think is enough for the user to not make a mistake. Also just 'Del' seems more natural to me. What do you think?

Reply:
I still prefer `Cmd/Ctrl + Del` which is consistent with other shortcuts, meanwhile providing flexibility for users to overwrite it in `.leptonrc`.

Reply:
@hackjutsu Sounds good. I've updated the shortcut to `Cmd/Ctrl + Del`. Thanks for your feedback.

Reply:
Could you help update this readme as well?

Reply:
@hackjutsu Sure! done.